docs/samples/Microsoft.ML.Samples/Dynamic/DataOperations/BootstrapSample.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using System.Collections.Generic;
+using Microsoft.ML.Data;
+
+namespace Microsoft.ML.Samples.Dynamic
+{
+    public static class BootstrapSample
+    {
+        public static void Example()
+        {
+            // Create a new context for ML.NET operations. It can be used for exception tracking and logging, 
+            // as a catalog of available operations and as the source of randomness.
+            var mlContext = new MLContext();
+
+            // Get a small dataset as an IEnumerable and them read it as ML.NET's data type.
+            IEnumerable<SamplesUtils.DatasetUtils.BinaryLabelFloatFeatureVectorSample> enumerableOfData = SamplesUtils.DatasetUtils.GenerateBinaryLabelFloatFeatureVectorSamples(5);
```

Comment:
>IEnumerable<SamplesUtils.DatasetUtils.BinaryLabelFloatFeatureVectorSample> [](start = 12, length = 74)

var?

Reply:
The current style of these Samples is to use these really long names to be explicit about types....